src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const calculator = {
+    result: 0,
+    add(value) {
+      calculator.result += value;
```

Comment:
almost correct, you should use `this` 

```suggestion
      this.result += value;
```